### PR TITLE
Add simpler radio buttons and checkboxes

### LINKED ISSUE #210: Create coded version of the Radio buttons and Checkboxes
**Description**
We currently only have a placeholder image version of the radio buttons and checkboxes. This issue to to complete the work to create an actual coded, and responsive, version of the visual design wireframes (see layout below).

**To Do**
- [x] Create html and styles (responsive) based off of current structure
- [x] Run HTML Codesniffer and review for accessibility quality and make any necessary changes
- [x] Submit pull request when code is ready for review
- [ ] Once reviewed and merged into staging, run by @angel, @carodew, @colinpmacarthur for final approval
- [ ] Iterate/make changes and re-submit PR with any fixes prior to pushing to production

radio_buttons_ui_v1

checkboxes_ui_v1

### PR DESCRIPTION
**Pull request by maya**
This is another take at radio buttons :radio: and checkboxes :heavy_check_mark:

Some issues could not be resolved from #216 so I redid them based off of Lea Verou's custom checkboxes from her book, "CSS Secrets."

Also adding in the disabled state.

This will deprecate #216 and resolves #210.

This is what it looks like:

screen shot 2015-07-23 at 8 25 21 pm

### CODE REVIEW

File: assets/_scss/elements/_inputs.scss [hunk 1]
```diff
@@ -57,4 +57,79 @@ label {
...
+input[type="checkbox"]:focus + label::before {
+  box-shadow: 0 0 0 1px $color-white, 0 0 0 3px $color-primary;
+}
+
+input[type="checkbox"]:disabled + label::before,
+input[type="radio"]:disabled + label::before {
+  background: $color-grey-600;
+  color: $color-grey-600;
+  cursor: not-allowed;
 }
```
Comment (juliaelman) [inline assets/_scss/elements/_inputs.scss]: New line at the end of file.

File: assets/_scss/elements/_typography.scss [hunk 2]
```diff
@@ -116,7 +116,7 @@ dfn {
 }
 
 .usa-sans,
-.usa-sans p {
+.usa-sans * {
```
Comment (juliaelman) [inline assets/_scss/elements/_typography.scss]: Why are we using a wildcard here? Not sure this is best practice and would be hard for users to un-tangle.

Comment (msecret) [inline assets/_scss/elements/_typography.scss]: We could change this so its all text elements, `<a>`, `<p>`, `<h1>`, etc. Or we could change the code so you add the class to the actual element you want to modify rather then the parent (which would be more verbose but safer.

File: assets/_scss/elements/_inputs.scss [hunk 3]
```diff
@@ -57,4 +57,79 @@ label {
...
+}
+
+/**
+ * Custom checkboxes
+ */
+
+input[type="checkbox"],
+input[type="radio"] {
+  position: absolute;
+  clip: rect(0,0,0,0);
```
Comment (msecret) [inline assets/_scss/elements/_inputs.scss]: Clip is deprecated:

File: assets/_scss/elements/_typography.scss (hunk 2 above)
Comment (maya) [inline assets/_scss/elements/_typography.scss]: What about lists and spans? Should we add them too? I didn't catch them so added the wildcard.

File: assets/_scss/elements/_inputs.scss [hunk 4]
```diff
@@ -57,4 +57,79 @@ label {
...
+  margin-bottom: 0.5em;
+}
+
+input[type="checkbox"] + label::before,
+input[type="radio"] + label::before {
+  box-shadow: 0 0 0 1px $color-grey-600;
+  border-radius: $usa-border-radius;  
+  content: '\a0';
+  display: inline-block;
+  vertical-align: .2em;
```
Comment (msecret) [inline assets/_scss/elements/_inputs.scss]: This have to be alphabetically ordered.

File: assets/_scss/elements/_typography.scss (hunk 2 above)
Comment (maya) [inline assets/_scss/elements/_typography.scss]: Updated in 54fdb72.

File: assets/_scss/elements/_inputs.scss (hunk 1 above)
Comment (maya) [inline assets/_scss/elements/_inputs.scss]: Done in 6034a4b.

File: assets/_scss/elements/_inputs.scss (hunk 4 above)
Comment (maya) [inline assets/_scss/elements/_inputs.scss]: Done in 9e55d3d.

File: assets/_scss/elements/_inputs.scss (hunk 3 above)
Comment (maya) [inline assets/_scss/elements/_inputs.scss]: Updated in d34550c and implemented in f7b58f9.

**Comment by juliaelman** [comment]
LGTM!

**Pull request merged**